guide/src/features.md

```diff
@@ -36,6 +36,10 @@ This feature changes [`Python::with_gil`]({{#PYO3_DOCS_URL}}/pyo3/struct.Python.
 
 If you do not enable this feature, you should call `pyo3::prepare_freethreaded_python()` before attempting to call any other Python APIs.
 
+### `anyhow-integration`
+
+This feature makes it possible to return [`anyhow::Result<T>`](https://docs.rs/anyhow/1.0.43/anyhow/type.Result.html) from functions and methods exposed to Python. It does so by adding `impl From<anyhow::Error> for PyErr`. Currently, the conversion simply stringifies the `anyhow::Error` and shoves it into a `PyRuntimeError`. As a consequence, there is no way to convert a `PyErr` back to the original `anyhow::Error`. It is mostly intended as a developer convenience.
```

Comment:
We should probably make the section with `num_bigint` etc into an `Optional Dependencies` section and add `anyhow` there.

Reply:
Are you saying I should rename the 'Advanced features' section in features.md to 'Optional features'?